README.md

```diff
@@ -24,15 +24,16 @@ To further configure your plugin, use the following custom parameters in your `s
-| `apiKMSKey`          | Datadog API Key encrypted using KMS. Use this parameter in place of `apiKey` when `flushMetricsToLogs` is `false`, and you are using KMS encryption.                                                                                                                                                                                                                                             |
-| `addLayers`          | Whether to install the Datadog Lambda library as a layer. Defaults to `true`. Set to `false` when you plan to package the Datadog Lambda library to your function's deployment package on your own so that you can install a specific version of the Datadog Lambda library ([Python][4] or [Node.js][5]). |
+| `flushMetricsToLogs` | Send custom metrics by using logs with the Datadog Forwarder Lambda function (recommended). Defaults to `true`. If you disable this parameter, it's required to set `apiKey` (or `apiKMSKey` if encrypted). Do not set `enableDDExtension` when using `flushMetricsToLogs`.                                                                                                                                                            |
+| `site`               | Set which Datadog site to send data, this is only used when `flushMetricsToLogs` is `false` or `enableDDExtension` is `true`. Possible values are `datadoghq.com`, `datadoghq.eu`, `us3.datadoghq.com` and `ddog-gov.com`. The default is `datadoghq.com`.                                                                                                                                                                                                                                |
+| `apiKey`             | Datadog API Key, only needed when `flushMetricsToLogs` is `false` or `enableDDExtension` is `true`. For more information about getting a Datadog API key, see the [API key documentation][3].                                                                                                                                                                                                                                    |
+| `apiKMSKey`          | Datadog API Key encrypted using KMS. Use this parameter in place of `apiKey` when `flushMetricsToLogs` is `false` or `enableDDExtension` is `true`, and you are using KMS encryption.                                                                                                                                                                                                                                             |
+| `addLayers`          | Whether to install the Datadog Lambda library or Datadog Lambda Extension as a layer. Defaults to `true`. Set to `false` when you plan to package the Datadog Lambda library to your function's deployment package on your own so that you can install a specific version of the Datadog Lambda library ([Python][4] or [Node.js][5]). |
```

Comment:
Does setting `addLayers` have an effect on how we install the extension? I thought this parameter was just for the regular layer but I might be wrong.

Reply:
As of right now if addLayers is false and addExtension is true then the extension will not be added because the addLayers parameter acts like a on/off switch for all layers although after thinking about it I think addLayers should just be regular layers and addExtension is its separate thing.

Reply:
Per our slack convo, addLayers and addExtension are now independent of another. The addLayers parameter strictly adds the lambda library as a layer while addExtension strictly adds the lambda extension as a layer.